deps/userns_sandbox.c

```diff
@@ -431,7 +431,15 @@ static void mount_the_world(const char * root_dir,
     // Create tmpfs to store ephemeral changes.  These changes are lost once
     // the `tmpfs` is unmounted, which occurs when all processes within the
     // namespace exit and the mount namespace is destroyed.
-    check(0 == mount("tmpfs", "/proc", "tmpfs", 0, "size=1G"));
```

Comment:
Yeah, it would be better for us to have `tmpfs_size_value` be a command-line value.

To do so, you first add the option to this list:
https://github.com/staticfloat/Sandbox.jl/blob/cad5200ae8c004249294fd9044495d17c8f37ad9/deps/userns_sandbox.c#L626-L635

Next, you add a `case` statement for the short argument name, and parse out the given value, like this:
https://github.com/staticfloat/Sandbox.jl/blob/cad5200ae8c004249294fd9044495d17c8f37ad9/deps/userns_sandbox.c#L729-L731

Then, you pass the new variable (say, `tmpfs_size`) to `mount_the_world()` and use that here.

Reply:
Done!